src/tokenizer/main.py

```diff
@@ -85,7 +85,70 @@
 group.add_argument(
     "--json", help="Output one token per line in JSON format", action="store_true"
 )
-group.add_argument("--normalize", help="Normalize punctuation", action="store_true")
+
+parser.add_argument(
+    "--s",
```

Comment:
These need to be changed to single hyphen forms: `-s`

Reply:
Done!